internal/pkg/inject/offset_results.json

```diff
@@ -2750,6 +2750,168 @@
+                      "1.14.4",
+                      "1.14.5",
+                      "1.14.6",
+                      "1.14.7",
+                      "1.14.8",
+                      "1.14.9",
+                      "1.14.10",
+                      "1.14.11",
+                      "1.14.12",
+                      "1.14.13",
+                      "1.14.14",
+                      "1.14.15",
+                      "1.15.0",
+                      "1.15.1",
+                      "1.15.2",
+                      "1.15.3",
+                      "1.15.4",
+                      "1.15.5",
+                      "1.15.6",
+                      "1.15.7",
+                      "1.15.8",
+                      "1.15.9",
+                      "1.15.10",
+                      "1.15.11",
+                      "1.15.12",
+                      "1.15.13",
+                      "1.15.14",
+                      "1.15.15",
+                      "1.16.0",
+                      "1.16.1",
+                      "1.16.2",
+                      "1.16.3",
+                      "1.16.4",
+                      "1.16.5",
+                      "1.16.6",
+                      "1.16.7",
+                      "1.16.8",
+                      "1.16.9",
+                      "1.16.10",
+                      "1.16.11",
+                      "1.16.12",
+                      "1.16.13",
+                      "1.16.14",
+                      "1.16.15",
+                      "1.17.0",
+                      "1.17.1",
+                      "1.17.2",
+                      "1.17.3",
+                      "1.17.4",
+                      "1.17.5",
+                      "1.17.6",
+                      "1.17.7",
+                      "1.17.8",
+                      "1.17.9",
+                      "1.17.10",
+                      "1.17.11",
+                      "1.17.12",
+                      "1.17.13",
+                      "1.18.0",
+                      "1.18.1",
+                      "1.18.2",
+                      "1.18.3",
+                      "1.18.4",
+                      "1.18.5",
+                      "1.18.6",
+                      "1.18.7",
+                      "1.18.8",
+                      "1.18.9",
+                      "1.18.10",
+                      "1.19.0",
+                      "1.19.1",
+                      "1.19.2",
+                      "1.19.3",
+                      "1.19.4",
+                      "1.19.5",
+                      "1.19.6",
+                      "1.19.7",
+                      "1.19.8",
+                      "1.19.9",
+                      "1.19.10",
+                      "1.19.11",
+                      "1.19.12",
+                      "1.19.13",
+                      "1.20.0",
+                      "1.20.1",
+                      "1.20.2",
+                      "1.20.3",
+                      "1.20.4",
+                      "1.20.5",
+                      "1.20.6",
```

Comment:
This seems broken: https://cs.opensource.google/go/go/+/refs/tags/go1.20.6:src/net/http/request.go;l=106